lib/game.js

```diff
@@ -29,16 +29,21 @@ star.src = "star.png";
 
 let ufo = new Ufo({x: 230, y: 60, width: 50, height: 20, isFlying: false});
 
-let topObstacle = new Obstacle({x: canvas.width, y: 100, width: 50, height: 50});
-let bottomObstacle = new Obstacle({x: canvas.width, y: canvas.height-200, width: 50, height: 50});
-let topObstacle_2 = new Obstacle({x: canvas.width+600, y: 200, width: 50, height: 50});
-let bottomObstacle_2 = new Obstacle({x: canvas.width+600, y: canvas.height-300, width: 50, height: 50});
-let topObstacle_3 = new Obstacle({x: canvas.width+600, y: 0, width: 50, height: 50});
-let bottomObstacle_3 = new Obstacle({x: canvas.width+600, y: canvas.height-75, width: 50, height: 50});
-let topObstacle_4 = new Obstacle({x: canvas.width, y: 0, width: 50, height: 50});
-let bottomObstacle_4 = new Obstacle({x: canvas.width, y: canvas.height-75, width: 50, height: 50});
-let topObstacle_5 = new Obstacle({x: canvas.width+600, y: 0, width: 50, height: 50});
-let bottomObstacle_5 = new Obstacle({x: canvas.width+600, y: canvas.height-75, width: 50, height: 50});
+const obstacleWidth = 50
```

Comment:
There's a bunch of stuff in here that isn't relevant to your refactor. Try to isolate changes in your commits.